saleor/graphql/webhook/types.py

```diff
@@ -107,8 +109,10 @@ class Meta:
 
 
 class EventDelivery(ModelObjectType[core_models.EventDelivery]):
-    id = graphene.GlobalID(required=True)
-    created_at = graphene.DateTime(required=True)
+    id = graphene.GlobalID(required=True, description="The ID of Event Delivery")
+    created_at = graphene.DateTime(
+        required=True, description="creation time of event delivery."
```

Comment:
Nitpick: would be good to keep it consistent and start each description with a capital letter.